Scarb.toml

```diff
@@ -39,8 +39,9 @@ keywords = [
 ]
 
 [workspace.dependencies]
+assert_macros = "2.8.0"
 starknet = "2.8.0"
```

Comment:
```suggestion
assert_macros = "2.8.2"
starknet = "2.8.2"
```

Reply:
Great catch. Should we also add these do the changelog? I haven't seen entries for starknet upgrades - so I imagine by signaling that we've upgraded scarb is enough. WDYT?

Reply:
Signaling scarb feels enough, since the scarb version is pegged to the compiler one, and the compiler comes with it.